### DISCUSSION TITLE
mysql vs. nginx

### DISCUSSION THREAD

**Discussion by ArniSamano** [YetiForceCompany/YetiForceCRM#16490] [General]
In terms of performance in case of large tables switching to nginx will significantly improve performance?

**Comment by vovpff** [comment]
You are try to compare web server and database? This is different tools for different tasks

**Comment by ArniSamano** [comment]
whatever we do in crm it goes through webserver. Question is how those two Apache and Nginx may differ in terms of delivery. Again... I am talking about large traffic (more than 2k users per hour)
Each user generating 100 to 300 records per hour :-)

**Reply by vovpff** [reply]
I don't have experience with this webserver load. My system around 20 users and little bit tuned apache is enough

**Comment by apcloic** [comment]
@ArniSamano 
For such a huge project, I think you should really talk with Yetiforce team because I don't think there is community users that work with 2000 concurrent users.

**Comment by mariuszkrzaczkowski** [comment]
it requires an individual approach,
We launched a system with multi-millionth number of records and 2000 users working at the same time
it worked with nginx

**Comment by GarethHumbel** [comment]
> In terms of performance in case of large tables switching to nginx will significantly improve performance?

CRM traffic is limited to number of users, so probably no hundreds of thousands per minute, System needs tuning with focus on database  rather than on web server

**Reply by mariuszkrzaczkowski** [reply]
everything needs tweaking, nginx / php / DB / OS

**Comment by diderich** [comment]
I would suggest first reviewing your use-case for 2000+ concurrent users and adding 100-300 records per hour! Is a full-fledged CRM system really what you need? What processing to you intend to do with all the data? Do you need the sophistication of YetitForceCRM with all its advantages and drawbacks? Is YetiForceCRM really the right system for your task at hand?

**Reply by mariuszkrzaczkowski** [reply]
He asks the question, it apparently needs this system
You're asking stupid off-topic questions.
Want to help answer this topic
